released/packages/coq-ext-lib/coq-ext-lib.0.9.7/opam

```diff
@@ -15,5 +15,5 @@ remove: [
   ["rm" "-R" "%{lib}%/coq/user-contrib/ExtLib"]
 ]
 depends: [
-  "coq" {>= "8.7" < "8.8~"}
+  "coq" {>= "8.7"}
```

Comment:
please say `< "8.9~"`. I'd be happy to say "Coq is always backward compatible", but it is not the case.